components/script/dom/bindings/trace.rs

```diff
@@ -544,7 +544,7 @@ unsafe_no_jsmanaged_fields!(WebGLTextureId);
 unsafe_no_jsmanaged_fields!(WebGLVertexArrayId);
 unsafe_no_jsmanaged_fields!(WebGLVersion);
 unsafe_no_jsmanaged_fields!(WebGLSLVersion);
-unsafe_no_jsmanaged_fields!(Identities);
+unsafe_no_jsmanaged_fields!(Arc<Mutex<Identities>>);
```

Comment:
We should probably use `parking_lot::Mutex` here. Parking lot is already used in Servo, and it's technically better than the standard mutex, and the locking doesn't require `unwrap()`.